_docs/for-contributors/2017-01-01-building-from-source.md

```diff
@@ -79,5 +79,20 @@ Clone Daedalus repository and go to the root directory:
 
 Then run the following script: 
 
-    [nix-shell:~/cardano-sl]$ ./scripts/link-bridge.sh
-    [nix-shell:~/cardano-sl]$ npm install
+    [nix-shell:~/daedalus]$ ./scripts/link-bridge.sh
+    [nix-shell:~/daedalus]$ npm install
+
+### Running acceptance tests
+
+To run acceptance tests one first has to have cluster running. We can run cluster on our machine with:
+
+    [nix-shell:~/cardano-sl]$ ./scripts/launch/demo-with-wallet-api.sh
+
+Then navigate to daedalus repo and run tests with:
+
+    [nix-shell:~/daedalus]$ npm run hot-server
+    [nix-shell:~/daedalus]$ npm run test
```

Comment:
You didn't point out that this should be launched in separate terminals (this is not obvious, I guess)